common/src/main/java/com/microsoft/identity/common/internal/cache/ADALOAuth2TokenCache.java

```diff
@@ -69,7 +69,20 @@
 
     private List<IShareSingleSignOnState> mSharedSSOCaches;
 
+    protected ISharedPreferencesFileManager getISharedPreferencesFileManager() {
+        return mISharedPreferencesFileManager;
+    }
+
+    protected void setISharedPreferencesFileManager(ISharedPreferencesFileManager iSharedPreferencesFileManager) {
```

Comment:
Let's remove this and declare `ISharedPreferencesFileManager` as `final` -- initialize once in the constructor. This shouldn't be swappable.